ccm_web/client/src/index.tsx

```diff
@@ -22,15 +22,11 @@ root.render(
      <ThemeProvider theme={ccmTheme}>
       <SnackbarProvider maxSnack={3}>
         <BrowserRouter>
-          <Switch>
-            <Route exact={true} path='/access-denied'>
-              <AccessDenied {...helpContactProps} />
-            </Route>
-            <Route exact={true} path='/launch-error'>
-              <LaunchError {...helpContactProps} />
-            </Route>
-            <Route><App /></Route>
-          </Switch>
+          <Routes>
+            <Route path='/access-denied' element={<AccessDenied {...helpContactProps} />}/>
+            <Route path='/launch-error' element={<LaunchError {...helpContactProps} />}/>
```

Comment:
You could trigged the launch error from Safari(and/or Private Window). Since Safari block 3rd party cookies completely unless you relaxed the setting from default.

Reply:
good tip! might try that next time